apps/marketing/content/design-system.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Design System
+---
+
+# Design System
+
+At Documenso we aim to be a design driven company. We believe that design is not just about how things look, but also how they work. We want to make sure that our products are easy to use and intuitive. We also want to make sure that our products are consistent and that they look and feel like they belong together.
+
+To achieve this we have created our design system which contains our primitives, token and screens alongside branding assets.
+
+We're open-sourcing our design system so that you can see how we build our products and think about design as a whole.
```

Comment:
@fmerian @ElTimuro Do your thang 💅🏻

Reply:
Looks like its above your paygrade

Reply:
no doubt no doubt

Reply:
`/design` may be a better (simpler) URI?

Reply:
Yes, I love the idea! 

Just imagine, https://design.documen.so, sounds powerful 😮‍💨

Reply:
for SEO, it's best to keep it under the main `documenso.com` domain, e.g. `documenso.com/design` (although [design.documenso.com](https://design.documenso.com/)) 🤯